portfolio/src/main/webapp/script.js

```diff
@@ -15,16 +15,17 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+// function addRandomGreeting() {
```

Comment:
if you're commenting it out because you don't need it, it's a good idea to get rid of it. If you want to keep it around to reference later, remember that git tracks all of your changes so you'll be able to find this snippet in your history later on.